### ISSUE TITLE
v-model, events "input" or "update: model-value" do not work in source edit mode

### ISSUE THREAD

**Issue by anfisa08** [ckeditor/ckeditor5-vue#254]
Hello,

I'm using Vue 3, Vite, TS and "@ckeditor/ckeditor5-vue": "5.1.0", "@ckeditor/ckeditor5-html-support": "^38.1.0", "@ckeditor/ckeditor5- edit source code": "^38.1.0".

When I turn on source edit mode and change something, the component does not send an "input" or "update: model-value" event. But when I turn off source edit mode, both of these events are dispatched.

I tried to solve the problem using the updateEditorData method of the SourceEditing plugin as recommended here https://github.com/ckeditor/ckeditor5/issues/11008. I used a keypress listener event and called this method if the mode is on.

But if I use v-model and this method, the component starts sending "input" or "update: model value" events in a loop.

**Comment by Witoso** [comment]
Thanks for reporting, when you say loop do you mean an infinite loop? If yes, then it's sounds like an issue we recently fixed ckeditor/ckeditor5#14469 in the main repo (not released yet).

**Comment by anfisa08** [comment]
Yes, it looks like it's the same. Thank you, I'll wait for the fix.
